contracts/core/extensions/CoreAccounting.sol

```diff
@@ -211,30 +211,30 @@ contract CoreAccounting is
         // Declare interface variavle for vault
         IVault vault = IVault(state.vault);
 
-        // Call Vault contract to deattribute tokens to user
-        vault.decrementTokenOwner(
+        // Call Vault contract to withdraw tokens from Vault to user
+        vault.withdrawTo(
             _token,
-            msg.sender,
+            _to,
             _quantity
         );
 
-        // Call Vault to withdraw tokens from Vault to user
-        vault.withdrawTo(
+        // Call Vault contract to deattribute tokens from user
+        vault.decrementTokenOwner(
```

Comment:
Switched the order of `withdrawTo` and `decrementTokenOwner` because, like in deposit, we want to ensure the deposit/withdrawal goes through successfully first, and then increment/decrement the user.

Reply:
Nevermind, gonna switch this back, I see the `require` inside `#decrementTokenOwner` which needs to happen first